packages/validator/src/cli-validator/run-validator.js

```diff
@@ -160,10 +157,10 @@ async function runValidator(cliArgs, parseOptions = {}) {
     let originalFile;
     let input;
 
-    if (filesToValidate.length > 1) {
-      logger.info(
-        '\n    ' + chalk.underline(`Validation Results for ${validFile}:`)
-      );
+    if (filesToValidate.length > 1 && context.config.outputFormat !== 'json') {
+      logger.info('');
+      logger.info(chalk.underline(`Validation Results for ${validFile}:`));
+      logger.info('');
```

Comment:
Perhaps these three lines should be part of the unconditional output (i.e. using `console.log`). I don't know if the results would make sense without them, if someone wanted to turn off logs.